client/lcd/root.go

```diff
@@ -96,3 +103,100 @@ func createHandler(cdc *wire.Codec) http.Handler {
 
 	return r
 }
+
+// ServeSwaggerCommand will generate a long-running rest server
+// that exposes functionality similar to the ServeCommand, but it provide swagger-ui
+// Which is much friendly for further development
+func ServeSwaggerCommand(cdc *wire.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "lite-server",
```

Comment:
@HaoyangLiu Could you add the cmd to the `Gaia-lite` section of [clients.md](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/sdk/clients.md) docs ?